src/shared/components/palette/base-field/abstract-form-field.component.ts

```diff
@@ -16,7 +16,7 @@ export abstract class AbstractFormFieldComponent {
   @Input()
   parent?: FormContainer;
 
-  protected registerControl<T extends AbstractControl>(control: T, replace = false): AbstractControl {
+  public registerControl<T extends AbstractControl>(control: T, replace = false): AbstractControl {
```

Comment:
Why has this been made public? If it's for testing purposes, that necessity had already been removed.

Reply:
Well I made it public to test the method. What do you suggest keep it protected?

Reply:
Yes please. It was made protected because it's only ever called from within the component (and anything that extends this base component) and never externally.